src/Container/ApplicationFactory.php

```diff
@@ -77,19 +63,13 @@ public function __invoke(ContainerInterface $container)
             ? $container->get(RouterInterface::class)
             : new FastRouteRouter();
 
-        $finalHandler = ! empty($config['zend-expressive']['raise_throwables'])
-            ? $this->marshalNoopFinalHandler($container)
-            : $this->marshalLegacyFinalHandler($container, $config);
+        $delegate = $this->marshalNoopFinalHandler($container);
```

Comment:
I just reviewed the 1.1.0 deprecation changes. I guess that would make this PR the expressive 2.0.0 version. That would mean stuff like this can be removed directly if 1.1.0 deprecates this stuff.